### PR TITLE
Article component revisions

### PR DESCRIPTION
**Pull request by thomasthesecond**

### CODE REVIEW

File: sass/_mixins.scss [hunk 1]
```diff
@@ -461,11 +461,40 @@
...
+  display: block; // Fallback for non-webkit browsers
+  display: -webkit-box;
+  font-size: $font-size;
+  height: ($font-size * $line-height * $lines-to-show); // Fallback for non-webkit browsers
+  line-height: $line-height;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: $lines-to-show;
+}
```
Comment (JoeShep) [inline sass/_mixins.scss]: Is this a replacement for the clamp js package? If not, seems a little confusing to have two clamping processes in the same code base. Any thoughts, @jcreamer898?

Comment (JoeShep) [inline sass/_mixins.scss]: Otherwise, :+1:

Comment (jcreamer898) [inline sass/_mixins.scss]: Agreed.

**Comment by thomasthesecond** [comment]
I can see where there'd be confusion. The reason I tried a CSS method was because that POI data is being loaded dynamically and the JS method to clamp seemed to work half the time. Not sure if it was because of the dynamically added content or not. Also, I thought a CSS-only solution would be better for progressive enhancement. But anyway, I think we just decided to truncate on the back-end, so I'll remove the mixin. Thanks for the feedback!

**Pull request merged**